.github/workflows/insitu.yml

```
@@ -30,3 +30,23 @@ jobs:
     - name: Build
       run: |
         cmake --build build -j 2
+  ascent:
```

Just some newline / spacing for better readability:
```suggestion

  ascent:
```